leasing/importer/area.py

```diff
@@ -534,11 +551,9 @@ def process_rows(
             if geom is None:
                 continue
 
-            update_data: UpdateData = {
-                "geometry": geom,
-                "metadata": metadata,
-                "external_id": row.id,
```

Comment:
This line was the one where the error originated from. With predetailed plans it gave an error, because the row doesn't have id. So I decided to create a method to conditionally add external_id when the area is a plan unit.

Reply:
As agreed in the previous meeting, let's add the `external_id` key with `row.id` value if row ID exists. Otherwise, let's leave out the `external_id` key entirely.

There might be no need for conditional logic about the AreaType, if the only condition will be whether the row ID exists or not.

Reply:
If the pattern holds for all plan units, then the area type might be useful to keep in our logic or at least mention in a comment. That way next time there is more information about the underlying data model and which fields are supposed to be there and which not.